Modules/Filtering/ImageCompare/test/CMakeLists.txt

```diff
@@ -38,3 +39,6 @@ itk_add_test(NAME itkTestingComparisonImageFilterTest
               ${ITK_TEST_OUTPUT_DIR}/itkTestingComparisonImageFilterTest.png
     itkTestingComparisonImageFilterTest DATA{${ITK_DATA_ROOT}/Input/cake_easy.png} DATA{${ITK_DATA_ROOT}/Input/cake_hard.png} ${ITK_TEST_OUTPUT_DIR}/itkTestingComparisonImageFilterTest.png 0 10 1 1647 11 66 16.2999 26846
 )
+itk_add_test(NAME itkImageFileReaderIOToRequestedRegionMismatchTest
+      COMMAND ITKImageCompareTestDriver itkImageFileReaderIOToRequestedRegionMismatchTest DATA{${ITK_DATA_ROOT}/Input/STAPLE1.png} DATA{${ITK_DATA_ROOT}/Input/BinaryImageWithVariousShapes01.png}
```

Comment:
This line is pretty long. Old lines 39 and 34 too. It would be good to break this command line into multiple CMake lines.

Reply:
Done. Amended the commit for the currently added test and done it for the existing ones in f1a7282.